### ISSUE TITLE
Support Svelte 4

### ISSUE THREAD

**Issue by silenceofthewans** [mattjennings/svelte-modals#31]
Changelog:
[https://github.com/sveltejs/svelte/blob/master/packages/svelte/CHANGELOG.md](https://github.com/sveltejs/svelte/blob/master/packages/svelte/CHANGELOG.md)

**Comment by matt8707** [comment]
Hi! Everything seems to work fine but npm is throwing this warning with Svelte 4. 
I think ´peerDependencies´ needs to be bumped. Thanks!

```bash
WARN  Issues with peer dependencies found
svelte-modals 1.2.1
✕ unmet peer svelte@^3.0.0: found 4.0.1
```

**Comment by mattjennings** [comment]
thanks for reporting this, should be fixed in 1.3.0
